lazy_tensor_core/lazy_tensor_core/csrc/ts_backend/aten_xla_type.cpp

```diff
@@ -836,6 +836,24 @@ at::Tensor permute(const at::Tensor& self, at::IntArrayRef dims) {
       LazyTensor::permute(self_tensor, Helpers::I64List(dims)));
 }
 
+at::Tensor repeat(const at::Tensor& self, at::IntArrayRef repeats) {
+  LTC_FN_COUNTER("xla::");
+  return bridge::AtenFromLtcTensor(LazyTensor::repeat(
+      bridge::GetLtcTensor(self), Helpers::I64List(repeats)));
+}
+
+// TODO(asuhan): Fix the select and copy_ combination below and rely on the core
```

Comment:
so this change takes us from 'broken' to 'slow-fallback but not a big deal, we can deal with it later'?  I'm not sure what cases this pattern occurs and how much that matters..

Reply:
Basically yes. In theory, if user code uses this pattern it'd still be broken too, but I've never seen it in the wild.